sdk/storage/storage-blob/src/utils/BuffersStream.ts

```diff
@@ -0,0 +1,56 @@
+import { Readable, ReadableOptions } from "stream";
+
+export interface BuffersStreamOptions extends ReadableOptions {}
+
+export class BuffersStream extends Readable {
```

Comment:
>BuffersStream [](start = 13, length = 13)

Add comments for class and internal properties/methods.

Reply:
Done.